api/package.json

```diff
@@ -12,18 +12,20 @@
   },
   "devDependencies": {},
   "dependencies": {
-    "@notum-cz/strapi-plugin-content-versioning": "^0.4.4",
     "@offset-dev/strapi-calendar": "^0.0.8",
-    "@strapi/plugin-i18n": "4.3.6",
+    "@strapi/plugin-graphql": "^4.4.1",
+    "@strapi/plugin-i18n": "4.4.1",
     "@strapi/plugin-seo": "^1.7.2",
-    "@strapi/plugin-users-permissions": "4.3.6",
-    "@strapi/strapi": "4.3.6",
-    "fs": "^0.0.1-security",
+    "@strapi/plugin-users-permissions": "4.4.1",
+    "@strapi/provider-email-sendmail": "^4.4.1",
```

Comment:
I don't think you actually need to declare this provider as it's added by default

Reply:
✅